bcda/client/bluebutton.go

```diff
@@ -103,10 +103,17 @@ func (bbc *BlueButtonClient) GetPatient(patientID, jobID, cmsID string) (string,
 	return bbc.getData(blueButtonBasePath+"/Patient/", params, jobID, cmsID)
 }
 
-func (bbc *BlueButtonClient) GetPatientByHICNHash(hashedHICN string) (string, error) {
+func (bbc *BlueButtonClient) GetPatientByIdentifierHash(hashedIdentifier string) (string, error) {
 	params := GetDefaultParams()
+
+	identifier := "hicnHash"
+	// will probably get rid of this by passing in an extra param
+	if utils.FromEnv("PATIENT_IDENTIFIER_MODE","HICN_MODE") == "MBI_MODE" {
+		identifier = "hicnMbi"
```

Comment:
Identifier should be `mbiHash` when in MBI mode.  See here: https://github.com/CMSgov/beneficiary-fhir-data/pull/172/files

Reply:
good eye!